build-system/server.js

```diff
@@ -149,14 +149,28 @@ var liveListCtr = 0;
 var itemCtr = 2;
 var liveListDoc = null;
 var doctype = '<!doctype html>\n';
-app.use(liveListUpdateFile, function(req, res) {
+// Only handle min/max
+app.use('/examples/live-list-update.amp.(min.|max.)html', function(req, res) {
+  var filePath = req.baseUrl;
+  var mode = getPathMode(filePath);
+  // When we already have state in memory and user refreshes page, we flush
+  // the dom we maintain on the server.
+  if (!('amp_latest_update_time' in req.query) && liveListDoc) {
+    var outerHTML = liveListDoc.documentElement./*OK*/outerHTML;
+    outerHTML = replaceUrls(mode, outerHTML);
+    res.end(`${doctype}${outerHTML}`);
```

Comment:
`res.send()` is better than `end()`.
it sets `content-type` header in response.